src/core/font_renderer.js

```diff
@@ -51,48 +66,49 @@ function getSubroutineBias(subrs) {
 
 function parseCmap(data, start, end) {
   const offset =
-    getUshort(data, start + 2) === 1
-      ? getLong(data, start + 8)
-      : getLong(data, start + 16);
-  const format = getUshort(data, start + offset);
+    getUint16(data, start + 2) === 1
+      ? getUint32(data, start + 8)
+      : getUint32(data, start + 16);
+  const format = getUint16(data, start + offset);
   let ranges, p, i;
   if (format === 4) {
-    getUshort(data, start + offset + 2); // length
-    const segCount = getUshort(data, start + offset + 6) >> 1;
+    getUint16(data, start + offset + 2); // length
+    const segCount = getUint16(data, start + offset + 6) >> 1;
     p = start + offset + 14;
     ranges = [];
     for (i = 0; i < segCount; i++, p += 2) {
-      ranges[i] = { end: getUshort(data, p) };
+      ranges[i] = { end: getUint16(data, p) };
     }
     p += 2;
     for (i = 0; i < segCount; i++, p += 2) {
-      ranges[i].start = getUshort(data, p);
+      ranges[i].start = getUint16(data, p);
     }
     for (i = 0; i < segCount; i++, p += 2) {
-      ranges[i].idDelta = getUshort(data, p);
+      ranges[i].idDelta = getUint16(data, p);
     }
     for (i = 0; i < segCount; i++, p += 2) {
-      let idOffset = getUshort(data, p);
+      let idOffset = getUint16(data, p);
       if (idOffset === 0) {
         continue;
       }
       ranges[i].ids = [];
       for (let j = 0, jj = ranges[i].end - ranges[i].start + 1; j < jj; j++) {
-        ranges[i].ids[j] = getUshort(data, p + idOffset);
+        ranges[i].ids[j] = getUint16(data, p + idOffset);
         idOffset += 2;
       }
     }
     return ranges;
   } else if (format === 12) {
-    getLong(data, start + offset + 4); // length
-    const groups = getLong(data, start + offset + 12);
+    // getUint32(data, start + offset + 4); // length
```

Comment:
If this isn't necessary any more, please don't leave commented-out code in the final patch.